include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -217,13 +217,6 @@ def TTIR_NegOp: TTIR_ElementwiseUnaryOp<"neg"> {
     }];
 }
 
-def TTIR_LogicalNotOp: TTIR_ElementwiseUnaryOp<"logical_not"> {
-    let summary = "Eltwise logical not op.";
-    let description = [{
-      Eltwise logical not operation.
-    }];
-}
-
```

Comment:
Is it just a weird diff or things were shuffled around?

Reply:
The list is growing so I alphabetically sorted the unary ops.

Reply:
FYI @sdjordjevicTT  who's also been doing some organization, but on the cpp files.